[2022-01-29 12:27:21,882] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-01-20T00:00:00+00:00 [queued]>
[2022-01-29 12:27:21,971] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-01-20T00:00:00+00:00 [queued]>
[2022-01-29 12:27:21,971] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:27:21,971] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-29 12:27:21,971] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:27:22,062] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-20 00:00:00+00:00
[2022-01-29 12:27:22,124] {standard_task_runner.py:52} INFO - Started process 3394 to run task
[2022-01-29 12:27:22,174] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-01-20T00:00:00+00:00', '--job-id', '257', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpepb4bj3j', '--error-file', '/tmp/tmp9_177ndh']
[2022-01-29 12:27:22,176] {standard_task_runner.py:77} INFO - Job 257: Subtask local_to_gcs_task
[2022-01-29 12:27:22,706] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-01-20T00:00:00+00:00 [running]> on host 85e1117cfb12
[2022-01-29 12:27:22,948] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-20T00:00:00+00:00
[2022-01-29 12:27:24,495] {python.py:175} INFO - Done. Returned value was: None
[2022-01-29 12:27:24,550] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20190120T000000, start_date=20220129T122721, end_date=20220129T122724
[2022-01-29 12:27:24,718] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 12:27:24,898] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
